src/components/table/helpers/mixin-tbody-row.js

```diff
@@ -237,7 +244,8 @@ export default {
               'aria-details': detailsId,
               'aria-owns': detailsId,
               'aria-rowindex': ariaRowIndex,
-              ...selectableAttrs
+              ...selectableAttrs,
+              ...additionalAttrs
```

Comment:
`additionalAttrs` should be before `selectableAttrs` so aria accessibility attributes do not get inadvertently overwritten